config/app/com/microsoft/azure/iotsolutions/uiconfig/services/external/PackageValidation/EdgePackageValidator.java

```diff
@@ -0,0 +1,9 @@
+package com.microsoft.azure.iotsolutions.uiconfig.services.external.PackageValidation;
+
+public class EdgePackageValidator extends PackageValidator{
+
+    @Override
+    public Boolean validate() {
+        return true;
```

Comment:
nit: add a comment that this will be updated

Reply:
added